float64 x_center
float64 y_center

float64 r

float64 a
float64 b
float64 angle

float64[] probabilities
int16 cls_num

